### PR TITLE
Fixed unopinionated tests to run on default viewport

### LINKED ISSUE #359: Unopinionated tests run on the last set/indexed viewport of Opinionated tests
# Bug Report

## Description

Unopinionated tests should run in **huge** viewport by default ([code here](https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraCommands/viewport-helpers.js#L34)) when formFactor has NOT been specified at command line/script.

## Steps to Reproduce

1. Goto terra-dialog-modal in terra-framework repo.
2. Delete the reference screenshots for "Huge" and "Large" viewport.
3. Run "npm run wdio" for the package.
4. Notice that only "large" viewport screenshots have been generated and "huge" viewport screenshots are not.
5. Also the unopinionated tests (as seen in the [spec file](https://github.com/cerner/terra-framework/blob/master/packages/terra-dialog-modal/tests/wdio/dialog-modal-spec.js)) have been tested against the "large" viewport

## Expected Behavior

The unopinionated tests should run on the huge viewport (default) rather than on the large viewport.

## Possible Solution

By reseting the viewport in the "after" method (as seen below) resolves this issue.
```
const describeViewports = (title, viewports, fn) => {
  let localViewports = viewports;

  // If formFactor is defined and viewports contains the form factor, run that size else don't run the tests
  const { formFactor } = global.browser.options;
  if (formFactor) {
    localViewports = viewports.includes(formFactor) ? [formFactor] : [];
  }
  localViewports.forEach(viewport => global.describe(`[${viewport}]`, () => {
    global.before(() => setViewport(viewport));
    global.describe(title, fn);
    global.after(() => setViewport(formFactor));
  }));
};
```

## @ Mentions

@emilyrohrbough

### PR DESCRIPTION
**Pull request by avinashg1994**
### Summary

Fixed Unopinionated tests to now run on huge viewport by default, rather than the last indexed viewport of Opinionated tests, when formFactor has NOT been specified at command line/script.

Resolves #359 

### Additional Details

A follow up issue should be created in the respective repos to cleanup the incorrect screenshots.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Unreleased
 ### Added
 * Add new terra docker containers based on alpine for ci and development
 
+### Fixed
+* Resetting the viewport to the form factor (if specified) at command line else to the default viewport (huge) for unopinionated tests.
```
Comment (emilyrohrbough) [inline CHANGELOG.md]:
```suggestion
* Restore the previous viewport size in the Terra. describeViewports helper
```

File: src/wdio/services/TerraCommands/viewport-helpers.js [hunk 2]
```diff
@@ -59,6 +59,7 @@ const describeViewports = (title, viewports, fn) => {
   localViewports.forEach(viewport => global.describe(`[${viewport}]`, () => {
     global.before(() => setViewport(viewport));
     global.describe(title, fn);
+    global.after(() => setViewport(formFactor));
```
Comment (emilyrohrbough) [inline src/wdio/services/TerraCommands/viewport-helpers.js]: This should restore the to the previous viewport size, not necessarily the formFactor defined in the test configuration

See: https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraCommands/axe-command.js#L87

Comment (avinashg1994) [reply src/wdio/services/TerraCommands/viewport-helpers.js]: Resolved this here : https://github.com/cerner/terra-toolkit/pull/363/commits/5cc921af7db23215348e91661e8cdf3e9597744c

File: package.json [hunk 3]
```diff
@@ -64,7 +64,7 @@
     "tt-wdio-webpack-obj": "node ./bin/wdio-runner.js wdio-runner-cli.js --suite opinionated",
     "tt-wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true node ./bin/wdio-runner.js --suite opinionated",
     "tt-wdio-unopinionated": "TT_TEST_WDIO_FUNCTION=true node  ./bin/wdio-runner.js --formFactors=['tiny','huge'] --locales=['en','fr'] --suite unopinionated;",
-    "tt-wdio-no-formFactor": "npm run pack && node ./bin/wdio-runner.js --spec tests/wdio/validateElement-spec.js && rm -rf ./build",
+    "tt-wdio-no-formFactor": "npm run pack && node ./bin/wdio-runner.js --spec tests/wdio/validateElement-spec.js --spec tests/wdio/defaultViewports-spec.js && rm -rf ./build",
```
Comment (emilyrohrbough) [inline package.json]: Only one spec will be run at a time. Are both tests being run with this change?

**Review by emilyrohrbough — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (avinashg1994) [reply package.json]: Yeah, my bad.
I changed it now to a suite including both the spec files.

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**